nvflare/apis/impl/task_controller.py

```diff
@@ -44,7 +44,12 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
     def start_controller(self, fl_ctx: FLContext):
         client_runner = fl_ctx.get_prop(FLContextKey.RUNNER)
```

Comment:
it would be good if we add some docstrings or the documentation saying this "TaskController" can only be configured/run on the client side.

Reply:
Actually this may not be true in the future.